cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -6109,6 +6109,73 @@ action.cyences_notable_event_action.products = Linux
 action.cyences_notable_event_action.teams = SOC
 
 
+[Linux - Possible Access To Sudoers File]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 10 * * * *
+description = This alert triggers when sudoers file will change. It controls user permissions for executing commands hence it could compromise the security of the host.
+dispatch.earliest_time = -61m@m
+dispatch.latest_time = -1m@m
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_linux_auditd` type=PATH name="/etc/sudoers*" \
+| stats count latest(_time) as _time by host name nametype OGID type tcontext_user inode \
+| rename name as dir_name nametype as dir_type OGID as group tcontext_user as user inode as unique_file_identifier \
+| `cs_human_readable_time_format(_time, event_time)` \
+| eval cyences_severity = if(count>2, "critical", "high") \
+| `cs_linux_access_to_sudoers_files_filter`
+action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.param.filter_macro_name = cs_linux_access_to_sudoers_files_filter
+action.cyences_notable_event_action.contributing_events = `cs_linux_auditd` type=PATH name="/etc/sudoers*"
+action.cyences_notable_event_action.system_compromised_search = | stats count by host
+action.cyences_notable_event_action.system_compromised_drilldown = `cs_linux_auditd` type=PATH name="/etc/sudoers*" host=$row.host$ 
+action.cyences_send_email_action = 1
+action.cyences_notable_event_action.products = Linux
+action.cyences_notable_event_action.teams = SOC
+
+
+[Linux - Disabled System Firewall]
```

Comment:
Linux - System Firewall Service Stopped

Update the macro name according to this as well.